portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,94 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/*
+ In order to access the secret page you have to solve a randomly generated equation.
+ Upon the input of a correct answer a button will appear, linking the user to the page.
+ */
+function secretPage(){
+  //Generates a number between 1 and 200 for both numbers.
+  this.a = Math.floor((Math.random()*200)+100);
+  this.b = Math.floor((Math.random()*200)+100);
+
+
+  //If b is a greater number we flip the values
+  if(this.b > this.a){
+    var hold =this.b;
+    this.b = this.a;
+    this.a = hold;
+  }
+
+  // This section of code includes the possible equations to solve
+  this.div = function(){
+    return this.a / this.b;
+  }
+  this.sub = function(){
+    return this.a - this.b;
+  }
+  this.add = function(){
+    return this.a + this.b;
+  } 
+  this.mul = function(){
+    return this.a * this.b;   
+  }
+  
+  //Checks if the answer givin is correct
+  let isCorrect = false;
+  
+  //Generates which equation option we choose;
+  let choice = Math.floor((Math.random()*4));
+  
+  
+  let input = 0;
+  let answer = 0;
+
+//choice determines which equation the user gets 
+  switch(choice){
+      case 0:
+        answer = this.add();
+        input = prompt(this.a +' + ' + this.b + '?',0);
+        if(input == answer){ 
```

Comment:
Here's a fun shorthand to collapse these three lines into one:

isCorrect = input == answer;

Reply:
Cool I definitely implement that.